packages/cli/src/commands/serve.ts

```diff
@@ -56,6 +57,7 @@ class DevelopmentServer {
                 ],
                 define: {
                     'process.env.NODE_ENV': JSON.stringify('development'),
+                    'DevCustomBlock.packages.appBridge': getAppBridgeVersion(process.cwd()),
```

Comment:
@SamuelAlev not sure if we should adjust it to `DevCustomBlock.<port>.packages.appBridge`, so you could serve multiple blocks with different appBridge versions